data_management/tests/initdb.py

```diff
@@ -285,7 +285,7 @@ def init_db(test=True):
     ExternalObject.objects.create(
         updated_by=user,
         object=o_paper,
-        doi_or_unique_name='10.15586/jptcp.v27iSP1.691',
+        other_unique_name='10.15586/jptcp.v27iSP1.691',
```

Comment:
This should be an `identifier` I think:
```suggestion
        identifier='https://dx.doi.org/10.15586%2Fjptcp.v27iSP1.691',
```